Magento_Theme/templates/html/social-list.phtml

```diff
@@ -0,0 +1,42 @@
+<ul class="list list--horizontal list--center">
+    <li class="list__item">
+        <a class="button button--link button--icon"
+           title="<?php /* @escapeNotVerified */echo __('Facebook') ?>"
+           href="#"
```

Comment:
I'll try to use snowdog/module-menu but currently there are an issue with the menu module #29436